src/AzureTopicConfigurationStrucuture.json

```diff
@@ -68,9 +68,5 @@
   {
     "topicName": "user_joined",
     "subscription": [ "Activity_UserJoinedMessageProcessor" ]
-  },
-  {
-    "topicName": "payment_created",
-    "subscription": [ "Activity_PaymentCreatedMessageProcessor" ]
```

Comment:
If we remove this topic and, as unlikely as it is, build a new environment the topic won't exist and the publication of messages will throw an exception.

Reply:
Removing the config will prevent the subscription from being created if devops delete it. We can leave the topic there though. That should work?